### PR TITLE
Fixed top-level readme w/ more detailed guideline

### PR DESCRIPTION
**Pull request by adizere**
Closes: #93 

## Description
______

For contributor use:

- [ ] Unit tests written
- [ ] Added test to CI if applicable 
- [ ] Updated CHANGELOG_PENDING.md
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation (`docs/`) and code comments
- [ ] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: relayer/cli/src/commands/start.rs [hunk 1]
```diff
@@ -114,7 +114,7 @@ async fn create_client(
 
     let store = relayer::store::persistent(format!("store_{}.db", chain.id())).unwrap(); //FIXME: unwrap
     let trust_options = store.get_trust_options().unwrap(); // FIXME: unwrap
-
+
```
Comment (ancazamfir) [inline relayer/cli/src/commands/start.rs:117]: I think there's a "whitechange" there, i.e. a tab
```suggestion


```

File: README.md [hunk 2]
```diff
@@ -35,11 +58,34 @@ configure and spawn two light clients for this chain with the following commands
...
+
+To set up a local development environment, clone the relayer implementation from [iqlusioninc/relayer](https://github.com/iqlusioninc/relayer/).
+We are interested in two commands we can run from this repo:
+
+- `bash scripts/two-chainz "local" "skip"`. Running this script will instantiate two chains, listening on ports `26557` and `26657`, respectively.
+
+
+- `bash dev-env`. Running this script script from your local source will instantiate two chains, on ports `26557` and `26657`, and also a relayer that sets up some connections and channels.
+
+Note that these script rely on the [cosmos/gaia](https://github.com/cosmos/gaia) implementation, which provides the basic tools for setting up a IBC-enabled Tendermint chain (called a "Cosmos Hub" in the IBC terminology).
```
Comment (ancazamfir) [inline README.md:86]:
```suggestion
Note that these script rely on the [cosmos/gaia](https://github.com/cosmos/gaia) implementation, which is a Cosmos-SDK application for the cosmos hub. 
```

File: README.md [hunk 3]
```diff
@@ -35,11 +58,34 @@ configure and spawn two light clients for this chain with the following commands
 4. Start the light clients and a dummy relayer thread:
 
     ```bash
-    ibc-rs $ cargo run --bin relayer -- -c ./relayer/relay/tests/config/fixtures/relayer_conf_example.toml start
+    ibc-rs $ cargo run --bin relayer -- -c ./relayer/relay/tests/config/fixtures/relayer_conf_example.toml start --reset
     ```
 
+    The `--reset` flag only needs to be passed once, for initializing the trusted headers based on the hash & height stored from steps 1-3 above.
+
+Beside the basic relayer `start` command, other subcommands are also supported at the moment:
```
Comment (ancazamfir) [inline README.md:66]:
```suggestion
Beside the basic relayer `start` command, the following are also available:
```

File: README.md [hunk 4]
```diff
@@ -35,11 +58,34 @@ configure and spawn two light clients for this chain with the following commands
 4. Start the light clients and a dummy relayer thread:
```
[the remark this answers is not in the document]
Comment (adizere) [reply README.md:58]: I'm not really sure what you mean by "event information". Do you mean the first two lines in the following log output?

```
Jun 15 17:55:00.601  INFO relayer::event_handler: Chain chain_B pushed {"NewBlock":{"height":"14057"}}
Jun 15 17:55:01.082  INFO relayer::event_handler: Chain chain_A pushed {"NewBlock":{"height":"14058"}}
....
Jun 15 19:46:21.524  INFO relayer_cli::commands::start: client updated to trusted state: 690C0969D66E6A4C378608A0AC7AE098AA4CDA20B87F363C276EACFF94FD2FE7 at height 20167 chain.id=chain_A
Jun 15 19:46:21.528  INFO relayer_cli::commands::start: client updated to trusted state: 26F4894926EE2A769737634239340CED649A64EDF9DF8926C9FAB892577F26C3 at height 20166 chain.id=chain_B
```

[the remark this answers is not in the document]
Comment (ancazamfir) [reply README.md:58]: If you run with `-v` you will see the events received by the relayer thread from the monitors.

**Comment by ancazamfir** [comment]
can you run `cargo fmt --all -- --check` and fix the failure. I think there's an extra blank like there.

**Pull request merged**